### PR TITLE
Makes Ling a bit better

### PR DESCRIPTION
**Pull request by Geevies**
Wew. It also separates ling stuff into sorted files and folders. For future developers to uh. Rework. And uh. Not remove.

* Improved sting descriptions.
* Fixed various typos, made some descriptions more clear.
* Stealthsuck now does toxins instead of genetic damage (randomly between 10 and 15).
* Hallucination sting occur 5 - 15 after stinging.
* Hallucinations last a smaller amount of time.
* Epinephrine sacs renamed to Adrenaline Sacs.
* Adrenaline Sacs cost 30 chemicals (was 45) previously.
* Adrenaline Sacs produce low amounts of oxycodone and hyperzine on activation.
* Faster chemical regeneration upgrade now regenerates 2.5 times faster (up from 2).
* Enhanced chemical storage now stores 50 extra (up from 25).
* Armblades now cost 4 genomes (up from 2).
* Rapid regeneration now heals genetic damage as well.
* Regenerative stasis now allows you to revive after 10 seconds (from randomly: between 8 and 20 seconds).
* Adds a text prompt confirmation if you want to go horrorform or not. Die misclicks, die.
* Adds a prompt to let ling know they need to wait a minute before they can stealthsucc again.
* Ling shield does more bashing damage.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by XanderDox** [comment]
What is the point of the rename? Epinephrine and Adrenaline are different names for the same chemical.

Why do the newly renamed Adrenaline Sacs produce two chemicals that are not... Adrenaline. We specifically have 'Epinephrine' (which is Adrenaline), in our code.

It seems like 'Chemical Sacs' is a more accurate name.

**Comment by XanderDox** [comment]
Also concerned that the Ling's most basic defensive weapon (swords) now cost more, despite being unable to pry open doors anymore, and being able to be disarmed out of Lings hands.

**Comment by Geevies** [comment]
> 
> 
> What is the point of the rename? Epinephrine and Adrenaline are different names for the same chemical.
> 
> Why do the newly renamed Adrenaline Sacs produce two chemicals that are not... Adrenaline. We specifically have 'Epinephrine' (which is Adrenaline), in our code.
> 
> It seems like 'Chemical Sacs' is a more accurate name.

Epinephrine sounds more clinical, while Adrenaline sounds more biological. It produces those chemicals because it'll fit for game balance. It isn't chemical sacs because the changeling already has a resource called "chemical", which this ability does not touch.

> 
> 
> Also concerned that the Ling's most basic defensive weapon (swords) now cost more, despite being unable to pry open doors anymore, and being able to be disarmed out of Lings hands.

The ling's most basic defensive weapon is not being caught. It costs more because it's better now. It can pry doors, because I was the one that made it able to do so. Being able to be disarmed out of the ling's hand is an unfortunate byproduct of how SS13 works, I can do nothing about it, and it is has nothing to do with this PR.

**Comment by XanderDox** [comment]
Should add epinephrine to the oxy-hyperzine mix then, we have it in the code and it's what the sac is named after.

**Comment by Geevies** [comment]
I see no point in doing that.

**Comment by XanderDox** [comment]
Because the name is entirely inaccurate and confusing when we have epinephrine in the code, but the epinephrine sacs aren't using it.

It would legitimately **make sense** to have in the sacs as well, since it is a stabilization chemical.

**Comment by Geevies** [comment]
I see.

File: code/game/gamemodes/changeling/implements/powers/suck.dm [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+		feedback_add_details("changeling_powers","A[stage]")
+		if(!do_mob(src, T, 75))
+			to_chat(src, "<span class='warning'>Our extraction of [T] has been interrupted!</span>")
+			changeling.isabsorbing = FALSE
+			return
+
+	to_chat(src, "<span class='notice'>We have finished infesting [T], and withdraw from their flesh, taking some of their genetic data.</span>")
+	to_chat(src, "<span class='notice'>We will need a minute to process this sample, before we can begin absorbing another.</span>")
+
+	addtimer(CALLBACK(T, /mob/living/.proc/adjustToxLoss, rand(10, 15)), rand(10, 15) SECONDS)
```
Comment (Geevies) [inline code/game/gamemodes/changeling/implements/powers/suck.dm]: addtimer(CALLBACK(T, /mob/living/.proc/add_reagent, "toxin", rand(10, 15)) rand(3, 5) SECONDS)

File: code/game/gamemodes/changeling/implements/powers/suck.dm [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+		feedback_add_details("changeling_powers","A[stage]")
+		if(!do_mob(src, T, 75))
+			to_chat(src, "<span class='warning'>Our extraction of [T] has been interrupted!</span>")
+			changeling.isabsorbing = FALSE
+			return
+
+	to_chat(src, "<span class='notice'>We have finished infesting [T], and withdraw from their flesh, taking some of their genetic data.</span>")
+	to_chat(src, "<span class='notice'>We will need a minute to process this sample, before we can begin absorbing another.</span>")
+
+	addtimer(CALLBACK(T, /mob/living/.proc/add_reagent, "toxin", rand(10, 15)) rand(3, 5) SECONDS)
```
Comment (Geevies) [inline code/game/gamemodes/changeling/implements/powers/suck.dm]:
```suggestion
	addtimer(CALLBACK(T.reagents, /mob/living/.proc/add_reagent, "toxin", rand(10, 15)) rand(3, 5) SECONDS)
```

File: code/game/gamemodes/changeling/implements/powers/suck.dm [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+		feedback_add_details("changeling_powers","A[stage]")
+		if(!do_mob(src, T, 75))
+			to_chat(src, "<span class='warning'>Our extraction of [T] has been interrupted!</span>")
+			changeling.isabsorbing = FALSE
+			return
+
+	to_chat(src, "<span class='notice'>We have finished infesting [T], and withdraw from their flesh, taking some of their genetic data.</span>")
+	to_chat(src, "<span class='notice'>We will need a minute to process this sample, before we can begin absorbing another.</span>")
+
+	addtimer(CALLBACK(T.reagents, /mob/living/.proc/add_reagent, "toxin", rand(10, 15)) rand(3, 5) SECONDS)
```
Comment (Geevies) [inline code/game/gamemodes/changeling/implements/powers/suck.dm]:
```suggestion
	T.reagents.add_reagent("toxin", rand(10, 15))
```

**Pull request merged**